configs/projects/agent-runtime-1.10.x.rb

```diff
@@ -7,6 +7,17 @@
   proj.setting :rubygem_net_ssh_version, '4.1.0'
   proj.setting :rubygem_semantic_puppet_version, '0.1.2'
 
+  platform = proj.get_platform
+
+  # This modification to the platform definition is for compatibility with
+  # pre-puppet6 puppet-agent packaging. Fedora 26 and 27 must have have an `f`
+  # prefix before their version numbers and do not use an explicit .dist
+  # string.
+  if platform.name =~ /^fedora-2(6|7)/
```

Comment:
Do you want this limited to 26 and 27? iirc, we're adding fedora 28 rn and that will need to keep the f for everything pre-puppet6 if we want to maintain consistency in the other streams.

Reply:
Ah, that makes sense -- I'll update this, thanks